src/color-picker/color-picker.scss

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+@import '../button/borderless';
+
+$iui-color-field-size: $iui-baseline * 19;
+$iui-color-swatch-box-shadow: inset 0 0 0 $iui-xxs - 1 
+  rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+$iui-color-swatch-border-radius: 5px;
+
+@mixin iui-color-picker {
+  @include iui-dropdown;
+  max-width: ($iui-l * 10) + ($iui-s * 9) + ($iui-sm * 2);
+  // max-width: (10 swatches per row) + (gaps) + (container padding)
+  max-height: ($iui-l * 10) + ($iui-s * 9) + ($iui-baseline * 2);
+  // max-height: (10 rows) + (gaps) + (container padding)
+  box-sizing: border-box;
+  border-radius: $iui-border-radius;
+  display: flex;
+  padding: $iui-baseline $iui-sm;
+  user-select: none;
+  overflow-y: auto;
+  @supports (overflow-y: overlay) {
+    overflow-y: overlay;
+  }
+
+  &.iui-advanced {
+    max-width: ($iui-color-field-size) + ($iui-sm) + ($iui-s) + ($iui-s) + ($iui-sm * 2);
+    // max-width: (color field width) + (color slider margins) + (color slider width) + (container padding);
+    flex-wrap: wrap;
+  }
+}
+
+@mixin iui-color-swatch {
+  height: $iui-l;
+  width: $iui-l;
+  cursor: pointer;
+  border-radius: $iui-color-swatch-border-radius;
+  background-color: var(--swatch-color);
+  margin-bottom: $iui-xs;
+  margin-right: $iui-xs;
+
+  @supports (gap: $iui-s) {
+    margin-bottom: 0;
+    margin-right: 0;
+  }
+
+  @include themed {
+    box-shadow: $iui-color-swatch-box-shadow;
+  }
+
+  &.iui-active {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1 t(iui-color-foreground-primary),
+        $iui-color-swatch-box-shadow;
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5)),
+        $iui-color-swatch-box-shadow;
+    }
+  }
+
+  &:focus {
+    outline: none;
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs + 1
```

Comment:
For focus and hover states, 3px spread is not too bad I guess, mainly because of the low opacity.

But maybe you could also try playing around with the blur-radius (the 3rd value) and share screenshots with results.
```suggestion
      box-shadow: 0 0 1px $iui-xxs + 1
```

Reply:
Changed blur-radius for focus box shadow 
Before & After:
![image](https://user-images.githubusercontent.com/88331924/131726162-f7880cb4-7df8-4121-ae47-fe180946303f.png)
![image](https://user-images.githubusercontent.com/88331924/131726172-e1e91bd3-d364-464d-82f0-a94f12e7859e.png)

Reply:
I cleaned it up to hopefully make it more clear, but I am trying to make the hover over active color thicker than the hover over non-active color because active color also has the active box shadow on it

Reply:
Why not keep 3px spread and 1px blur in all focus/hover states (including active focus)? It's simpler and looks ok in all cases.
![focus](https://user-images.githubusercontent.com/9084735/131742506-c2fafef9-8488-441e-be14-122e55017555.gif)

Reply:
I changed the focus and hover shadows to 3px and 2px

Reply:
Done

Reply:
You forgot the `themed` mixin so I doubt it's even working. Also you should put the whole `.iui-active` block after the base hover/focus states to help the cascade.

Reply:
I added the themed mixin

Reply:
The only thing missing now is the `iui-focus` mixin on color dot

Reply:
Should I remove $iui-focus-box-shadow and $iui-hover-box-shadow from the color dot and have only the iui-focus mixin? Or keep both?

Reply:
Keep hover, replace the `:focus` block with `@include iui-focus`

Reply:
Ok added

Reply:
Ah wait, I see you have a custom box shadow on the dot too, so the mixin will not work (it overrides the iui-elevation shadow). You will need to manually define `focus:not(:focus-visible)` style, similar to how you did for swatch.

Or I guess revert to the previous commit, it was not too bad.

Reply:
Ok I reverted it